Source/Core/Core/ScriptEngine.cpp

```diff
@@ -21,12 +22,17 @@
 #include "Core/PowerPC/MMU.h"
 #include "Core/PowerPC/PowerPC.h"
 
+// clang-format off
+#define DOLPHIN_LUA_METHOD(x) {.name = #x, .func = (x)}
```

Comment:
Would it make sense to move this further down to the table that uses it? Plus, at least something I like to do for local macros like this one; `#undef` it after use?

Reply:
Thanks, good point. Fixed